doc/source/releases/0.10.0.rst

```diff
@@ -0,0 +1,72 @@
+========================
+Neo 0.10.0 release notes
+========================
+
+27th July 2021
+
+
+New IO modules
+--------------
+
+.. currentmodule:: neo.io
+
+* :class:`CedIO` - an alternative to :class:`Spike2IO`
+* :class:`AxonaIO`
+* :class:`OpenEphysIO` - handle the binary format
+* :class:`PhyIO`
+* :class:`SpikeGLXIO`
+* :class:`NWBIO` - support for a subset of the `NWB:N`_ format
+* :class:`MaxwellIO`
+
+
+Bug fixes and improvements in IO modules
+----------------------------------------
+
+* :class:`NeuralynxIO` was refactored and now supports new file versions (neuraview) and single file loading.
+* Legacy versions of old IOs were removed for NeuralynxIO (neuralynxio_v1), BlackrockIO, NeoHdf5IO.
+* :class:`NixIOfr` now supports array annotations of :class:`AnalogSignal` objects.
+* The IO modules of this release have been tested with version 0.1.0 of the `ephy_testing_data`_.
+* :class:`NSDFIO` was removed because we can no longer maintain it.
+* all IOs now accept :class:`pathlib.Path` objects.
```

Comment:
I would move the housekeeping note about the version of the test files to the end of the list
```suggestion
* :class:`NSDFIO` was removed because we can no longer maintain it.
* all IOs now accept :class:`pathlib.Path` objects.
* The IO modules of this release have been tested with version 0.1.0 of the `ephy_testing_data`_.
```